simpegEM/FDEM/SurveyFDEM.py

```diff
@@ -16,6 +16,20 @@ class RxFDEM(Survey.BaseRx):
                     'bxi':['b', 'Fx', 'imag'],
                     'byi':['b', 'Fy', 'imag'],
                     'bzi':['b', 'Fz', 'imag'],
+
+                    'jxr':['j', 'Fx', 'real'],
```

Comment:
no, we should be careful here. right now Problem_e and Problem_b can take e, b receivers and Problem_j and Problem_h can take j,h receivers. Is there a better way to implement this?